src/Symfony/Component/Finder/Finder.php

```diff
@@ -440,6 +440,22 @@ public function sortByName(bool $useNaturalSort = false): static
         return $this;
     }
 
+    /**
+     * Sorts files and directories by name case insensitive.
+     *
+     * This can be slow as all the matching files and directories must be retrieved for comparison.
+     *
+     * @return $this
+     *
+     * @see SortableIterator
+     */
+    public function sortByCaseInsensitiveName(bool $useNaturalSort = false): static
```

Comment:
What about adding a second argument to the existing `sortByName()` method instead?